### PR TITLE
Add migration instructions to deprecation messages

### PR DESCRIPTION
**Pull request by Shashi456**
Fix: #386.

### CODE REVIEW

File: Sources/TensorFlow/BackwardsCompatibility.swift [hunk 1]
```diff
@@ -200,7 +200,11 @@ public extension Conv1D where Scalar.RawSignificand: FixedWidthInteger {
     ///
     /// - Note: Use `init(filterShape:stride:padding:dilation:activation:seed:)` for faster random
     ///   initialization.
-    @available(*, deprecated, message: "This API will be removed after Swift for TensorFlow 0.4.")
+    @available(*, deprecated, message: """
+    This API will be removed after Swift for TensorFlow 0.4, please consider using
```
Comment (eaplatanios) [inline Sources/TensorFlow/BackwardsCompatibility.swift]:
```suggestion
        This API will be removed after Swift for TensorFlow 0.4, please consider using
```

File: Sources/TensorFlow/BackwardsCompatibility.swift [hunk 2]
```diff
@@ -200,7 +200,11 @@ public extension Conv1D where Scalar.RawSignificand: FixedWidthInteger {
...
+    'init(filterShape:stride:padding:dilation:activation:filterIntializer:biasInitializer:)'
```
Comment (eaplatanios) [inline Sources/TensorFlow/BackwardsCompatibility.swift]:
```suggestion
        `init(filterShape:stride:padding:dilation:activation:filterIntializer:biasInitializer:)`
```

File: Sources/TensorFlow/BackwardsCompatibility.swift [hunk 3]
```diff
@@ -200,7 +200,11 @@ public extension Conv1D where Scalar.RawSignificand: FixedWidthInteger {
...
+    instead.
```
Comment (eaplatanios) [inline Sources/TensorFlow/BackwardsCompatibility.swift]:
```suggestion
        instead.
```

File: Sources/TensorFlow/BackwardsCompatibility.swift [hunk 4]
```diff
@@ -200,7 +200,11 @@ public extension Conv1D where Scalar.RawSignificand: FixedWidthInteger {
...
+    """)
```
Comment (eaplatanios) [inline Sources/TensorFlow/BackwardsCompatibility.swift]:
```suggestion
        """)
```

File: Sources/TensorFlow/BackwardsCompatibility.swift [hunk 5]
```diff
@@ -221,18 +225,22 @@ public extension Conv1D where Scalar.RawSignificand: FixedWidthInteger {
...
     /// - Parameters:
-    ///   - filterShape: The 3-D shape of the filter, representing	
+    ///   - filterShape: The 3-D shape of the filter, representing
     ///   - padding: The padding algorithm for convolution.
     ///   - dilation: The dilation factor for the temporal dimension.
     ///   - activation: The element-wise activation function.
     ///   - seed: The random seed for initialization. The default value is random.
-    @available(*, deprecated, message: "This API will be removed after Swift for TensorFlow 0.4.")
+    @available(*, deprecated, message: """
+    This API will be removed after Swift for TensorFlow 0.4, please consider using
```
Comment (eaplatanios) [inline Sources/TensorFlow/BackwardsCompatibility.swift]: Same suggested changes as above.

File: Sources/TensorFlow/BackwardsCompatibility.swift [hunk 6]
```diff
@@ -271,7 +279,11 @@ public extension Conv2D {
     ///
     /// - Note: Use `init(filterShape:strides:padding:activation:seed:)` for faster random
     ///   initialization.
-    @available(*, deprecated, message: "This API will be removed after Swift for TensorFlow 0.4.")
+    @available(*, deprecated, message: """
+    This API will be removed after Swift for TensorFlow 0.4, please consider using
```
Comment (eaplatanios) [inline Sources/TensorFlow/BackwardsCompatibility.swift]: Same suggested changes as above.

File: Sources/TensorFlow/BackwardsCompatibility.swift [hunk 7]
```diff
@@ -307,7 +319,11 @@ public extension Conv2D {
     ///     (dilation height, dilation width).
     ///   - activation: The element-wise activation function.
     ///   - seed: The random seed for initialization. The default value is random.
-    @available(*, deprecated, message: "This API will be removed after Swift for TensorFlow 0.4.")
+    @available(*, deprecated, message: """
+    This API will be removed after Swift for TensorFlow 0.4, please consider using
```
Comment (eaplatanios) [inline Sources/TensorFlow/BackwardsCompatibility.swift]: Same suggested changes as above.

File: Sources/TensorFlow/BackwardsCompatibility.swift [hunk 8]
```diff
@@ -345,7 +361,10 @@ public extension Conv3D {
     ///
     /// - Note: Use `init(filterShape:strides:padding:activation:seed:)` for faster random
     ///   initialization.
-    @available(*, deprecated, message: "This API will be removed after Swift for TensorFlow 0.4.")
+    @available(*, deprecated, message: """
+    This API will be removed after Swift for TensorFlow 0.4, please consider using
```
Comment (eaplatanios) [inline Sources/TensorFlow/BackwardsCompatibility.swift]: Same suggested changes as above.

File: Sources/TensorFlow/BackwardsCompatibility.swift [hunk 9]
```diff
@@ -378,7 +397,10 @@ public extension Conv3D {
     ///   - padding: The padding algorithm for convolution.
     ///   - activation: The element-wise activation function.
     ///   - seed: The random seed for initialization. The default value is random.
-    @available(*, deprecated, message: "This API will be removed after Swift for TensorFlow 0.4.")
+    @available(*, deprecated, message: """
+    This API will be removed after Swift for TensorFlow 0.4, please consider using
```
Comment (eaplatanios) [inline Sources/TensorFlow/BackwardsCompatibility.swift]: Same suggested changes as above.

File: Sources/TensorFlow/BackwardsCompatibility.swift [hunk 10]
```diff
@@ -412,7 +434,10 @@ public extension TransposedConv2D {
     ///
     /// - Note: Use `init(filterShape:strides:padding:activation:seed:)` for faster random
     ///   initialization.
-    @available(*, deprecated, message: "This API will be removed after Swift for TensorFlow 0.4.")
+    @available(*, deprecated, message: """
+    This API will be removed after Swift for TensorFlow 0.4, please consider using
```
Comment (eaplatanios) [inline Sources/TensorFlow/BackwardsCompatibility.swift]: Same suggested changes as above.

File: Sources/TensorFlow/BackwardsCompatibility.swift [hunk 11]
```diff
@@ -443,7 +468,10 @@ public extension TransposedConv2D {
     ///   - padding: The padding algorithm for convolution.
     ///   - activation: The element-wise activation function.
     ///   - seed: The random seed for initialization. The default value is random.
-    @available(*, deprecated, message: "This API will be removed after Swift for TensorFlow 0.4.")
+    @available(*, deprecated, message: """
+    This API will be removed after Swift for TensorFlow 0.4, please consider using
```
Comment (eaplatanios) [inline Sources/TensorFlow/BackwardsCompatibility.swift]: Same suggested changes as above.

File: Sources/TensorFlow/BackwardsCompatibility.swift [hunk 12]
```diff
@@ -476,7 +504,10 @@ public extension DepthwiseConv2D {
     ///
     /// - Note: Use `init(filterShape:strides:padding:activation:seed:)` for faster random
     ///   initialization.
-    @available(*, deprecated, message: "This API will be removed after Swift for TensorFlow 0.4.")
+    @available(*, deprecated, message: """
+    This API will be removed after Swift for TensorFlow 0.4, please consider using
```
Comment (eaplatanios) [inline Sources/TensorFlow/BackwardsCompatibility.swift]: Same suggested changes as above.

File: Sources/TensorFlow/BackwardsCompatibility.swift [hunk 13]
```diff
@@ -506,7 +537,10 @@ public extension DepthwiseConv2D {
     ///   - padding: The padding algorithm for convolution.
     ///   - activation: The element-wise activation function.
     ///   - seed: The random seed for initialization. The default value is random.
-    @available(*, deprecated, message: "This API will be removed after Swift for TensorFlow 0.4.")
+    @available(*, deprecated, message: """
+    This API will be removed after Swift for TensorFlow 0.4, please consider using
```
Comment (eaplatanios) [inline Sources/TensorFlow/BackwardsCompatibility.swift]: Same suggested changes as above.

**Comment by Shashi456** [comment]
@eaplatanios done.

**Comment by eaplatanios** [comment]
Thanks @Shashi456!

**Comment by Shashi456** [comment]
@saeta The toolchain would have to be updated with this commit since it adds the warning messages.

**Pull request merged**